### PR TITLE
Add transform tests

### PR DESCRIPTION
**Pull request by TrySound**
Ref #33

### CODE REVIEW

**Comment by TrySound** [comment]
Would be great to add travis to automate checking.

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "url": "https://github.com/d3/d3-zoom.git"
   },
   "scripts": {
-    "pretest": "rm -rf build && mkdir build && rollup --banner \"$(preamble)\" -g d3-dispatch:d3,d3-drag:d3,d3-interpolate:d3,d3-selection:d3,d3-transition:d3 -f umd -n d3 -o build/d3-zoom.js -- index.js",
-    "test": "tape 'test/**/*-test.js' && eslint index.js src",
+    "pretest": "rm -rf build && mkdir build && rollup -g d3-dispatch:d3,d3-drag:d3,d3-interpolate:d3,d3-selection:d3,d3-transition:d3 -f umd -n d3 -o build/d3-zoom.js -- index.js",
```
Comment (TrySound) [inline package.json]: Don't know how to run build.js with this string before

File: package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
...
+    "test": "tape test/**/*-test.js && eslint index.js src",
```
Comment (TrySound) [inline package.json]: Quotes doesn't play on windows.

**Comment by mbostock** [comment]
Your changes break somethings on macOS, so I can’t merge this. If there’s a way to fix Windows without breaking macOS please let me know. Specifically, the --preamble argument is required to generate the preamble in the bundles, and I don’t know how to pass a string with spaces without using quotes; and likewise the quotes are needed for tape so that the glob is evaluated by tape and not the shell.

**Comment by TrySound** [comment]
preamble command is not evaluated on windows bash too. So, I will just revert it before every commit.
Why do not let shell evaluate globs? Result is the same, isn't it?
Please reopen PR to let me finish my work.

**Comment by mbostock** [comment]
Sorry, I don’t know how to fix this on Windows without breaking it on macOS. Can’t let the shell evaluate the double star glob ** because it doesn’t work reliably in different shells; at least that was my experience in the past, and why I switched to using a quoted arg so that it could be globbed by tape.

I’ll reopen and merge this PR if you…

* revert the changes to the package.json (or discover how to fix the aforementioned problems on macOS)
* switch to two- (rather than four-) space indents
* switch to single (rather than double) quotes for strings in the test

Thank you!

**Comment by TrySound** [comment]
@mbostock Done. Please keep PRs opened caz new commits are not watched in closed.

**Comment by mbostock** [comment]
I can’t reopen since you force-pushed to that branch, apparently, but I’ve created a new PR #39. However, there’s still an issue. Please respond over there.

**Pull request closed without merging**